docs/install/quickstart.md

```diff
@@ -13,28 +13,36 @@ If NetworkManager is installed and enabled on your hosts, [ensure that it is con
 
 - For RKE2 versions 1.21 and higher, if the host kernel supports [AppArmor](https://apparmor.net/), the AppArmor tools (usually available via the `apparmor-parser` package) must also be present prior to installing RKE2.
 
+- The RKE2 installation process must be run as the root user or through `sudo`.
```

Comment:
I initially thought to add `# Run as root user or through sudo` on every single code block but it gets tiresome quickly. Instead, I prefered to add this requirement here and on L61, then added `sudo` install in the comments when seems needed